listener/src/event_handler/order_handler.rs

```diff
@@ -29,7 +29,7 @@ impl EventHandler for SellOrderHandler {
         info!(logger, "Processing SellOrder {:?}", &order);
 
         let mut entity: Entity = order.into();
-        entity.set("id", &entity_id);
+        entity.set("id", format!("{}_0", &entity_id));
```

Comment:
<a href="https://gitme.me/image?url=https%3A%2F%2Fmedia2.giphy.com%2Fmedia%2F3ohzdYJK1wAdPWVk88%2Fgiphy.gif&token=oops" data-gitmeme-token="oops"><img src="https://media2.giphy.com/media/3ohzdYJK1wAdPWVk88/giphy.gif" title="Created by gitme.me with /oops"/></a>

Reply:
@fleupold this was on purpose. 

https://github.com/gnosis/dex-services/pull/211/commits/adbfcec07d4747d734f22668576de3ff901d32c5

The thing is the standing orders use an id of: `block_hash + log_index + order number`

For coherence between all orders, I decided to follow a naming convention, I just add a `_0` for normal orders, since they are the first and only order.

I don't mind to remove it, I don't have a strong opinion about it.

Reply:
Ok I didn't see that. I'm fine with that.